app/src/pages/inside/dashboardItemPage/modals/widgetWizardModal/widgetWizardContent/utils.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import _ from 'lodash';
```

Comment:
Why did you decide to use lodash here?
We already use `fast-deep-equal` for equality checks.
I'm not sure some of our dependencies use it in production, so the importing full lib may affect our bundle size.